cpp/include/proton/target_options.hpp

```diff
@@ -60,6 +60,10 @@ class target_options {
     /// address is ignored if dynamic() is true.
     PN_CPP_EXTERN target_options& address(const std::string& addr);
 
+    /// Set the target be of type coordinator.
+    /// This immediately override the currently assigned type.
+    PN_CPP_EXTERN target_options& make_coordinator();
+
```

Comment:
I don't think this is needed to use the transaction API. If it's used in the implementation it needs to be purely internal. But I don't think it's actually used at all.